### DISCUSSION TITLE
option for disallowing users to send private and channel messages

### DISCUSSION THREAD

**Discussion by amirramezani79** [BearWare/TeamTalk5#1934] [Ideas]
Hi.
When creating/editing users, We can prevent users from voice , video and media transmition.
I suggest to add option for private and channel messages to be disabled for a user.

**Comment by bear101** [comment]
@CoBC should there also be a USERRIGHT_TEXTMESSAGE_USER and USERRIGHT_TEXTMESSAGE_CHANNEL ?

**Comment by CoBC** [comment]
Yes, it's a good idea to reduce spam on some servers.
